pkgs/development/ocaml-modules/ogg/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildDunePackage, fetchFromGitHub, dune-configurator, libogg }:
+
+buildDunePackage rec {
+  pname = "ogg";
+  version = "0.7.0";
+
+  useDune2 = true;
+
+  src = fetchFromGitHub {
+    owner = "savonet";
+    repo = "ocaml-ogg";
+    rev = "v${version}";
+    sha256 = "02gw8vbmmvnjnwixhy07m9fpaddmgj3kfwf2h7264gxr4lp1krgg";
+  };
+
+  buildInputs = [ dune-configurator ];
+  propagatedBuildInputs = [ libogg ];
+
+  meta = with lib; {
+    homepage = "https://github.com/savonet/ocaml-ogg";
+    description = "Bindings to libogg";
+    license = licenses.gpl2Only;
```

Comment:
The license is LGPL 2.1 (https://github.com/savonet/ocaml-ogg/blob/v0.7.0/COPYING). BTW, 0.7.1 is out.